Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonManager.cs

```diff
@@ -10,6 +10,8 @@
  */
 #endregion
 
+using System;
```

Comment:
Why is this here ?

Reply:
Think it has been added by ReSharper, removed now